index.html

```diff
@@ -14,31 +14,22 @@
 
 <body>
     <header>
-        <div class="gutter-right logo">
-            <img src="assets/big-logo.png"/>
-        </div>
+        <h1>Pokemon Catcher</h1>
     </header>
     <main>
-        <section class="main-section">
-            <h2>Pokemon</h2>
-            <div id="three-pokemon">
-                <label>
-                    <input type="radio" name="pokemon"/>
-                    <img />
-                </label>
-                <label>
-                    <input type="radio" name="pokemon"/>
-                    <img />
-                </label>
-                <label>
-                    <input type="radio" name="pokemon"/>
-                    <img />
-                </label>
-            </div>
-            <div id="next" class="hidden">
-                <div id="capture"></div>
-                <button>Next</button>
-            </div>
+        <section id="main-section">
+            <label>
+                <input type="radio" name="pokemon"/>
+                <img />
+            </label>
+            <label>
+                <input type="radio" name="pokemon"/>
+                <img />
+            </label>
+            <label>
+                <input type="radio" name="pokemon"/>
+                <img />
+            </label>
```

Comment:
also great work transferring over your logic to your radios.  Personally, I think it makes the user experience more streamlined by having it function on click